payments-core/src/main/java/com/stripe/android/payments/bankaccount/navigation/CollectBankAccountResult.kt

```diff
@@ -0,0 +1,29 @@
+package com.stripe.android.payments.bankaccount.navigation
+
+import android.os.Parcelable
+import com.stripe.android.core.model.StripeModel
+import kotlinx.parcelize.Parcelize
+
+/**
+ * The result of an attempt to collect a bank account
+ */
+sealed class CollectBankAccountResult : Parcelable {
+
+    @Parcelize
+    data class Completed(
+        val response: CollectBankAccountResponse
+    ) : CollectBankAccountResult()
+
+    @Parcelize
+    data class Failed(
+        val error: Throwable
```

Comment:
Since this error message is not localized we should make it clear when exposing publicly that it is a developer message, not one intended for the user (not one that is recoverable).